src/main/java/mil/dds/anet/resources/PositionResource.java

```diff
@@ -169,10 +169,8 @@ public Integer updatePosition(@GraphQLRootContext Map<String, Object> context,
   public int updatePositionHistory(@GraphQLRootContext Map<String, Object> context,
       @GraphQLArgument(name = "position") Position pos) {
     final Person user = DaoUtils.getUserFromContext(context);
-    final Position existing = dao.getByUuid(pos.getUuid());
-    assertCanUpdatePosition(user, existing);
 
-    final String existingPersonUuid = DaoUtils.getUuid(existing.getPerson());
+    final String existingPersonUuid = DaoUtils.getUuid(pos.getPerson());
```

Comment:
This is most certainly not correct; permissions should be checked against the *database* version, not against the version supplied by the user (as there they could fake things such that they do have permission). So the existing position and the assert are *necessary*.